src/components/Header.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <header>
+        <h1>{{title}}</h1>
+        <Button color="green" text="Add"/>
```

Comment:
What does this button do?

Reply:
This is toggle button. Still not fully functional
- commented